package.json

```diff
@@ -127,12 +140,19 @@
     "eslint-plugin-prettier": "^5.1.3",
     "eslint-plugin-react": "^7.33.2",
     "eslint-plugin-simple-import-sort": "^12.0.0",
+    "eslint-plugin-storybook": "^0.8.0",
     "eslint-plugin-tailwindcss": "^3.14.2",
     "eslint-plugin-unused-imports": "^3.1.0",
     "inquirer": "^9.2.14",
     "postcss-import": "^16.0.1",
     "remark-cli": "^12.0.1",
     "remark-lint": "^10.0.0",
-    "remark-mdx": "^3.0.1"
+    "remark-mdx": "^3.0.1",
+    "storybook": "^8.1.5"
+  },
+  "eslintConfig": {
+    "extends": [
+      "plugin:storybook/recommended"
+    ]
   }
```

Comment:
Add the eslint config in the `eslintrc` file only.

Reply:
Done